src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2670,7 +2670,8 @@ def delete_ssl_cert(cmd, resource_group_name, certificate_thumbprint):
 
 def import_ssl_cert(cmd, resource_group_name, name, key_vault, key_vault_certificate_name):
     Certificate = cmd.get_models('Certificate')
-    client = web_client_factory(cmd.cli_ctx)
+    # TODO remove api_version when Microsoft.CertificateRegistration supports a later version
+    client = web_client_factory(cmd.cli_ctx, api_version="2021-01-15")
```

Comment:
May I ask why `Microsoft.DomainRegistration` and `Microsoft.CertificateRegistration` don't support the new api-version as `ResourceType.MGMT_APPSERVICE`?
If the supported api-version between them and `ResourceType.MGMT_APPSERVICE` are independent of each other, I suggest setting the sub configurations for them separately in the `_shared.py`, just like https://github.com/Azure/azure-cli/blob/b4759064b7e38203ef9d67a581105206a17f513b/src/azure-cli-core/azure/cli/core/profiles/_shared.py#L151-L164

Reply:
The App Service certs and domains back end team just released the new API version, so I just removed all the places where I was hardcoding the API version in the client

Reply:
> @StrawnSC May I ask why `Microsoft.DomainRegistration` and `Microsoft.CertificateRegistration` don't support the new api-version as `ResourceType.MGMT_APPSERVICE`? If the supported api-version between them and `ResourceType.MGMT_APPSERVICE` are independent of each other, I suggest setting the sub configurations for them separately in the `_shared.py`, just like
> 
> https://github.com/Azure/azure-cli/blob/b4759064b7e38203ef9d67a581105206a17f513b/src/azure-cli-core/azure/cli/core/profiles/_shared.py#L151-L164

@zhoxing-ms this was a temporary change, while we were waiting on ARM manifest update for the new API version to complete across all locations & get the initial code review unblocked.